storage/sealer/sched.go

```diff
@@ -139,6 +141,13 @@ type workerResponse struct {
 	err error
 }
 
+type rmRequest struct {
+	id     uuid.UUID
+	rmresE chan error
+	rmresC chan struct{}
+	Ctx    context.Context
```

Comment:
This isn't used. Also note that in Go it's contexts should only be passed through func params, saving them in structs is not the best practice (although there are some cases where that's necessary)

```suggestion
```